### DISCUSSION TITLE
Ecommerce

### DISCUSSION THREAD

**Discussion by JHAID-DEV** [notJust-dev/notjustdev#242] [General]
pls my name is success from Ghana an i am new to development and just a beginner a have many ideas of creating an ecommerce website wih instant messaging app with personal mobile payment but i don't know how to code so please help me achieve my dream

**Comment by JHAID-DEV** [comment]
Please i need your help @Savinvadim1312 @notjust-dev @Saad-Bashar @ItsLukas

**Comment by VadimNotJustDev** [comment]
What should I understand from this and how do you expect to be helped?

**Reply by JHAID-DEV** [reply]
sorry boss, I have an idea of building an ecommerce and instant messaging app website but i don't know how to code and I have the UI design please. please here is it. https://www.figma.com/design/r8UcCnHk8O4zyHxByNLTzy/Hichat---Messenger-App-UI-Kit-(Community)?m=auto&t=EYFroHljCOsebk9E-6 and https://www.figma.com/design/ihiWQN3tFf62AL9UhfCAdf/Kutuku----eCommerce-Mobile-App-Ui-Kits-(Community)?t=8IUzRoQPGrIUfjqn-1 and https://www.figma.com/design/TlO7kLrvaaH6uRTAI8vrcD/AllPay---Finance%2C-Banking%2C-%26-E-Wallet-App-UI-Kit-(Community)?t=EYFroHljCOsebk9E-1
